tutorials/analysis_3d.ipynb

```diff
@@ -592,7 +592,699 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.3"
+   "version": "3.7.0"
```

Comment:
Is it possible to avoid this extra output somehow? Maybe just edit the notebook json directly for now, without re-executing it? And then we ask @Bultako for help next week?